src/wristpy/io/readers/readers.py

```diff
@@ -13,9 +13,6 @@
 def read_watch_data(file_name: pathlib.Path | str) -> models.WatchData:
     """Read watch data from a file.
 
-    This function selects the correct loader based on the file extension.
-    Returns error if none of the above.
-
     Args:
         file_name: The filename to read the watch data from.
```

Comment:
I would add a little more information about the supported extensions here, just as a user convenience.

Reply:
I'll add one line briefly, but I assume this would be better covered by the readme for the package?

Reply:
yea totally, i was just imagining a quick line a user could glance at quickly, nothing super involved.